wrangler.toml

```diff
@@ -1,13 +1,11 @@
 name = "workers-graphql-server"
-type = "webpack"
 
 account_id = ""
-zone_id = ""
-route = ""
 
 workers_dev = true
-webpack_config = "webpack.config.js"
 compatibility_date = "2021-10-03"
```

Comment:
Updating this to current date can turn on new features and API's

Reply:
Thanks for the review Jaboc.
I am not sure about which compatibility date would be best.
So i put some comments linking to it and the opt in example.